pages/api/templates/[formID].ts

```diff
@@ -58,10 +59,15 @@ const templates = async (
     // If not GET then we're authenticated and can safely return the complete Form Record.
     return res.status(200).json(response);
   } catch (err) {
-    if (err instanceof AccessControlError) return res.status(403).json({ error: "Forbidden" });
-    else if (err instanceof TemplateAlreadyPublishedError)
+    if (err instanceof AccessControlError) {
+      return res.status(403).json({ error: "Forbidden" });
+    } else if (err instanceof TemplateAlreadyPublishedError) {
       return res.status(500).json({ error: "Can't update published form" });
```

Comment:
We need to be careful when using http error code 500. Our alarms look for this code to sense if anything is critically wrong with the application.
As a team we should try to ensure we stick with the appropriate http status response requests depending on what we're trying to convey.  
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses

Reply:
For example `Can't update published form` should probably be returned as a `409`

Reply:
I was trying to find a code that would make sense but I think I ended up on a website that was not giving that many options. I will use 409!